frontend/javascripts/components/async_clickables.js

```diff
@@ -57,10 +57,10 @@ export function AsyncIconButton(props: Props & { icon: React.Element<*> }) {
 export function AsyncLink(props: Props & { icon: React.Node }) {
   const [isLoading, onClick] = useLoadingClickHandler(props.onClick);
   const icon = isLoading ? <LoadingOutlined key="loading-icon" /> : props.icon;
-  const content = [icon, props.children];
```

Comment:
Array children need a key. This triggered a fairly verbose React warning, before.